js/clickup-add-issues/README.md

```diff
@@ -0,0 +1,20 @@
+# Clickup add issues
+
+This sequence reads info send by [git-read-issues]() then adds all the issues to a clickup list
+
+> ! This sequence should be used only in combination with git-read-issues
+
+## Setup
+
+Change data in `cudata.json` to one that applies to you
+
+```json
+{
+    "listId":"list-id",//id of the list where issues will be placed
+    "token":"token"//api-token
```

Comment:
can we move the token from the config file to the argument of the Sequence to be passed at the start-up? It would be aligned with our security guidelines for SCP. thanks :)

Reply:
Thank you, made all needed changes ;)